crates/api/src/site/purge/comment.rs

```diff
@@ -26,10 +28,10 @@ pub async fn purge_comment(
 
   let comment_id = data.comment_id;
 
-  // Read the comment to get the post_id
-  let comment = Comment::read(&mut context.pool(), comment_id).await?;
+  // Read the comment to get the post_id and community
+  let comment = CommentView::read(&mut context.pool(), comment_id, None).await?;
```

Comment:
Pry should rename to `cv` or `comment_view` for clarity.

Reply:
Done